src/app.ts

```diff
@@ -123,18 +123,22 @@ async function pollForStatus() {
     } else {
         if (pollResult.reconfigure) {
             if (!reconfigureLock) {
+                logger.info('Reconfigure detected, triggering reconfiguration');
                 // set reconfigure lock, only reconfigure if configure is not already running
                 reconfigureLock = true;
                 try {
+                    statsReporter.setReconfigureStart(pollResult.reconfigure);
                     await commandHandler.reconfigure();
-                    statsReporter.setReconfigureError(false);
+                    statsReporter.setReconfigureEnd(false);
```

Comment:
Shouldn't be set to true?  statsReporter.setReconfigureEnd(true);

Reply:
No we only set the error to true if we had a problem, in the catch case